docs-user/js/init.js

```diff
@@ -9,4 +9,5 @@ window.$docsify = {
   ga: 'UA-35433268-81',
   // Photon blue-50
   themeColor: '#0a84ff',
+  noCompileLinks: ['.*\\.svg']
```

Comment:
We link to SVG files in some places and this doesn't work. For example in https://profiler.firefox.com/docs/#/./guide-stack-samples-and-call-trees, you can search for the text `A useful exercise at this point would be to consider the running time and self time of each node this call tree.` and click on the links.